### DISCUSSION TITLE
unknown directive: toolchain

### DISCUSSION THREAD

**Discussion by telekomancer** [clouditor/clouditor#1223] [Q&A] [answered]
https://github.com/clouditor/clouditor/blob/0cf9593702896c8a30ccede0416d6a698ebb9fbb/go.mod#L4C1-L5C19

``` 
go install google.golang.org/protobuf/cmd/protoc-gen-go google.golang.org/grpc/cmd/protoc-gen-go-grpc github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway github.com/google/gnostic/cmd/protoc-gen-openapi
go: errors parsing go.mod:
~/clouditor/go.mod:5: unknown directive: toolchain
``` 

This is on a bare metal machine any help ?

**Comment by oxisto** [comment, marked as answer]
Do you have Go 1.21 installed? The toolchain directive requires at least Go 1.21. See https://go.dev/blog/toolchain

**Reply by telekomancer** [reply]
I tried to check on documentation but i don't see nothing related to go version, could be included on docs.

kinds!
